src/app/policies/components/PolicyConnections.vue

```diff
@@ -12,17 +12,17 @@
   >
 
   <DataSource
-    v-slot="{ data, isLoading, error }: PolicyDataplaneCollectionSource"
+    v-slot="{ data, error }: PolicyDataplaneCollectionSource"
     :src="`/meshes/${props.mesh}/policy-path/${props.policyPath}/policy/${props.policyName}/dataplanes`"
   >
-    <LoadingBlock v-if="isLoading" />
-
     <ErrorBlock
-      v-else-if="error"
+      v-if="error"
       :error="error"
     />
 
-    <EmptyBlock v-else-if="data === undefined || data.items.length === 0" />
+    <LoadingBlock v-else-if="data === undefined" />
+
+    <EmptyBlock v-else-if="data.items.length === 0" />
 
     <template v-else>
       <p
```

Comment:
Can we change this to at least an `<ul>` or something like this while we're touching this?

Reply:
Done.